README.rst

```diff
@@ -186,13 +193,33 @@ iterates over all courts of a certain jurisdiction that is provided by a
 script. See ``lib/importer.py`` for an example that's used in
 the sample caller.
 
+District Court Parser
+=====================
+A sample driver to run the PACER District Court parser on an html file is included.
+It takes HTML file(s) as arguments and outputs JSON to stdout.
+
+Example usage:
+
+::
+
+   PYTHONPATH=`pwd` juriscraper/pacerdocket.py tests/examples/pacer/dockets/district/nysd.html
+
+
 Tests
 =====
 
 We got that! You can (and should) run the tests with
 ``tox``. This will run ``python setup.py test`` for all supported Python runtimes,
 iterating over all of the ``*_example*`` files and run the scrapers against them.
 
+Individual tests can be run with:
+
+   python -m unittest tests.test_pacer.DocketParseTest.test_district_court_dockets
+
+Or, to run and drop to the Python debugger if it fails:
+
+  nosetests -v --pdb tests/test_pacer.py:DocketParseTest.test_district_court_dockets
```

Comment:
Nose isn't a dependency now. Perhaps add it to requirements-dev.txt if we want this trick?

Reply:
Well...no? It's not a dependency.  If you want to debug a test under pdb, it's great to know you can do it this way, but it's not required for the code to work to do development, so it doesn't qualify as a dependency?

Reply:
K. Can you tweak the comment then to say that you'll need to install nose? For people using virtualenvs, it's weird to get an instruction in the readme that has uninstalled requirements.

Reply:
Done.